tests/test_blocks.py

```diff
@@ -73,4 +73,5 @@ def test_block(block_details):
     assert (coin.header_prevhash(block.header)
             == hex_str_to_hash(block_info['previousblockhash']))
     for n, (tx, txid) in enumerate(block.transactions):
-        assert txid == hex_str_to_hash(block_info['tx'][n])
+        if n < len(block_info['tx']):
```

Comment:
This change needs an explanation

Reply:
I think when Decred mainnet was added they combined normal and staking transactions (which are separate fields when retrieving block info from dcrd) into the block.json's tx array. Without doing that for the testnet block.json, this loop would go out of range on the ```block_info['tx']``` array because the decoded ```block.transactions``` include both staking and normal tx's.